nimble/controller/src/ble_ll_ctrl.c

```diff
@@ -1584,7 +1584,7 @@ ble_ll_ctrl_rx_conn_update(struct ble_ll_conn_sm *connsm, uint8_t *dptr)
 
     /* Only a slave should receive this */
     if (connsm->conn_role == BLE_LL_CONN_ROLE_MASTER) {
-        return BLE_ERR_MAX;
+        return BLE_LL_CTRL_UNKNOWN_RSP;
```

Comment:
I think we should reply with `LL_REJECT_IND` since this is not an unknown PDU, it's just invalid in current role

Reply:
well and what error code would you used then?

On the other side you probably agree that this LL_CTRL_PDU is not used in the given role.
That said look at this from Bluetooth 5 spec:

"If an LL Control PDU is received that is not used or not supported, the Link
Layer shall respond with an LL_UNKNOWN_RSP PDU"

Reply:
so let's leave `LL_UNKNOWN_RSP` then - the quotation from spec is what I was missing here and could not find this myself

Reply:
that makes me encourage to add this sentence to commit message. Let me do it.

Reply:
done